kubectl-pg/LICENSE

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
```

Comment:
Why the plugin comes with a different license than the operator itself?

Reply:
I used cobra it has generated it. I need to do some cleanup didn't pay attention towards it. I will remove that.